nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/JoltTransformJSON.java

```diff
@@ -95,6 +99,24 @@
             .required(false)
             .build();
 
+    public static final PropertyDescriptor CUSTOM_CLASS = new PropertyDescriptor.Builder()
+            .name("jolt-custom-class")
+            .displayName("Custom Transformation Class Name")
+            .description("Fully Qualified Class Name for Custom Transformation  Module Directory should be specified")
```

Comment:
This looks like it's two sentences so should have a period in between. Also the second part could be elaborated upon, since the Module Directory shouldn't need to be specified (if their JAR is somehow already in the NiFi classpath), rather the class just needs to be able to be loaded by the processor by any means. Not sure how to word that...